build.gradle

```diff
@@ -271,7 +279,7 @@ project('client') {
         compile project(':shared:authplugin')
         compile project(':shared:protocol')
         compile project(":shared:controller-api")
-
+        compile project(":shared:security"), withoutJaxb
```

Comment:
I tried to remove the JAXB dependency from the `shared:security` module, but then realized it is necessary for the `io.jsonwebtoken` dependency. Without it, some of the tests fail (Issue #3937). 

Since Jwt helper in the module isn't used in the client, I've explicitly excluded Jaxb (as well as JJWT dependency now) module. 

Hope that clarifies.